tests/test_post_summary_parse.py

```
@@ -0,0 +1,228 @@
+
+def test_parse_roles_from_dict_entries_with_actor_types() -> None:
+    content = _json_content(
+        {
+            "roles_and_responsibilities": [
+                {"actor_name": "김다은", "responsibility": "검토"},
+                {"actor_name": "설계부", "responsibility": "승인", "actor_type": "organization"},
+                {"actor_name": "설계팀", "responsibility": "배포", "actor_type": "Team"},
+            ]
+        }
+    )
+    roles, projects = _parse_summary_details(content)
```

Ruff (0.16.2)

[warning] Unpacked variable `projects` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)